### PR TITLE
Improvement: add version createdAt range filter

### PR DESCRIPTION
**Pull request by Henya14**

### CODE REVIEW

File: core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java [hunk 1]
```diff
@@ -32,6 +32,12 @@
 	@Parameter(description = "The types of resources to get the versions for")
 	private List<String> resourceType;
 	
+	@Parameter(description = "Greater than equal to filter for the created at field")
+	private Long createdAtFrom;
```
Comment (cmark) [inline core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java:39]: Could you please also add the simple exact match query parameter `createdAt`?

**Review by cmark — APPROVED** [review]
LGTM!

File: core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java (hunk 1 above)
Comment (Henya14) [reply core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java:39]: Yes

Comment (Henya14) [reply core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java:39]: Done

**Review by cmark — APPROVED** [review]
LGTM!

**Pull request merged**